ext/web/lib.deno_web.d.ts

```diff
@@ -140,11 +140,22 @@ declare type EventListenerOrEventListenerObject =
   | EventListenerObject;
 
 interface AddEventListenerOptions extends EventListenerOptions {
+  /** A boolean value indicating that the listener should be invoked at most
+   * once after being added. If true, the listener would be automatically
+   * removed when invoked. */
   once?: boolean;
+  /** A boolean value that, if true, indicates that the function specified by
+   * listener will never call preventDefault(). If a passive listener does call
+   * preventDefault(), the user agent will do nothing other than generate a
```

Comment:
This might require some customization for Deno context. Scrolling performance will not really make sense for these event listeners, I would presume.